migrate_pim/apply_patch/apply_patch_ce.rst

```diff
@@ -0,0 +1,47 @@
+Guide to apply a patch for Community Edition
+=============================================
+
+Run the composer update command:
+
+.. code-block:: bash
+
+    composer --prefer-dist update
+
+Be aware that your composer.json won't be updated and some dependencies might be missing or from an outdated version.
+
+You have to check whether the latest composer.json is different from your own. In this case you should backup your current composer.json and download the newest one beforehand.
```

Comment:
"Is different from your own" .... I'd say: "from yours"

Reply:
:+1: